components/inputs/test/input-date-range.visual-diff.js

```diff
@@ -63,11 +63,14 @@ describe('d2l-input-date-range', () => {
 
 	async function getRectInnerTooltip(page, selector, inputDateSelector) {
 		return page.$eval(selector, (elem, inputDateSelector) => {
+			let content = window.querySelectorComposed(elem.shadowRoot, 'd2l-tooltip[showing]');
```

Comment:
If this works I will clean this up and use the same solution in the other two places that do the same thing. Unless of course there is a better approach. My goal here is to update the test so that it doesn't need to know which component is responsible for the tooltip. This method just needs to locate the tooltip being displayed so that it can calculate the correct dimensions for the screenshot.